litmus-portal/frontend/src/components/Header/index.tsx

```diff
@@ -175,13 +171,12 @@ const Header: React.FC = () => {
                 <CustomBreadCrumbs location={useLocation().pathname} />
               </Box>
               <Box p={1} className={classes.headerFlexPadded}>
-                {/* uncomment to activate notifications.
-                  <NotificationsDropdown
+                {/*              <NotificationsDropdown
                   count={`${countOfMessages}`}
                   messages={messages}
```

Comment:
You may remove this commented code

Reply:
This was intentionally left to keep some info about the props, interface and functions to be used when we have the data for notifications. It was also there from beforehand. Can be removed with another PR if the team agrees. @oumkale

Reply:
Remove the notifications comment, will add it later when the feature is ready

Reply:
this can be removed @ishangupta-ds

Reply:
Removed it.